components/typography/metadata/typography.yml

```diff
@@ -154,11 +294,62 @@ examples:
         <p class="spectrum-Body5">Article Body5 Text <em>Article Body5 Emphasis</em> <strong>Article Body5 Strong</strong>.</p>
         <p class="spectrum-Detail">Detail Text <em>Detail Emphasis</em> <strong>Detail Strong</strong>.</p>
       </div>
+  - id: detail-1
+    status: Verified
+    name: Detail
+    description: Subheadings for typography.
+    markup: |
+      <div class="spectrum">
+        <p class="spectrum-DetailXL">DetailXL <em class="spectrum-DetailXL--light">DetailXL Emphasis</em> <strong class="spectrum-DetailL--strong">DetailXL Strong</strong>.</p>
+        <p class="spectrum-DetailL">DetailL <em class="spectrum-DetailL--light">DetailL Emphasis</em> <strong class="spectrum-DetailL--strong">DetailL Strong</strong>.</p>
+        <p class="spectrum-DetailM">DetailL <em class="spectrum-DetailM--light">DetailM Emphasis</em> <strong class="spectrum-DetailL--strong">DetailM Strong</strong>.</p>
+        <p class="spectrum-DetailS">DetailL <em class="spectrum-DetailS--light">DetailS Emphasis</em> <strong class="spectrum-DetailL--strong">DetailS Strong</strong>.</p>
```

Comment:
Docs incorrectly label examples as "DetailL" for Medium and Small

Reply:
> It appears none of the font stacks are coming directly from DNA. This becomes evident in the fonts listed to use for Arabic and Hebrew, which differ from what's defined in DNA. I would like to discuss this file:
> 
> https://github.com/adobe/spectrum-css/blob/2f4fab674cc10f692a7ab3bcac799af5ad84a5fd/components/commons/fonts.css#L32
> 
> and how we can ensure that CSS pulls these fallback font stacks from DNA here: https://git.corp.adobe.com/Spectrum/spectrum-dna/blob/master/data/aliases/StaticAliases.mjs#L59

Just  to clarify: arabic & hebrew use myriad and the rest is going to use adobe clean in it's international versions?
I am not sure why those were hard-coded here, maybe @GarthDB  knows more about this.

Reply:
is there an option for `  --spectrum-font-family-base: 'adobe-clean-ux', 'adobe-clean', 'Source Sans Pro', var(--spectrum-font-fallbacks-sans);` ?